frontend/src/modules/contributor/components/edit/work-history/contributor-work-history-edit.vue

```diff
@@ -71,14 +76,19 @@
         <lf-button type="secondary-ghost" @click="close">
           Cancel
         </lf-button>
-        <lf-button
-          type="primary"
-          :loading="sending"
-          :disabled="$v.$invalid || sending"
-          @click="updateWorkExperience()"
+        <lf-tooltip
+          :disabled="!(hasSameOrgDetails && !isEdit)"
+          content="Please enter a different Job title or Period, as there is already a work experience associated with the selected organization."
         >
-          {{ isEdit ? 'Update' : 'Add' }} work experience
-        </lf-button>
+          <lf-button
+            type="primary"
+            :loading="sending"
+            :disabled="$v.$invalid || sending || (hasSameOrgDetails && !isEdit)"
```

Comment:
Could you remove !isEdit?
This should be applied when a user is adding a work experience or editing it